Cargo.toml

```
@@ -8,8 +8,8 @@ edition = "2018"
 
 [dependencies]
 # esp32 = { version = "0.2.1" }
-esp32 = { path = "../../hal/esp32-pac"}
+esp32 = { path = "../esp32"}
```

```suggestion
esp32 = { git = "https://github.com/esp-rs/esp32.git" }
```
and `cargo update` when necessary, until `esp32` gets released to crates.io.

---

I released 0.2.2 last night, https://crates.io/crates/esp32